python/psim/cli/config.py

```diff
@@ -0,0 +1,35 @@
+from .. import config
+
+import argparse
+import json
+
+
+def main(args):
+    
+    parser = argparse.ArgumentParser(
+        prog='psim config', description='Process PSim configuration files.'
+    )
+    parser.add_argument(
+        '--modules', '-m', nargs='+', required=False, type=str, help='Set of ' +
+        'modules to process configuration files for. Defaults to all modules ' +
```

Comment:
Should say "defaults to all modules _if_ none are specified"

Reply:
Yep this will change!